src/In.ProjectEKA.HipService/DataFlow/HiDataRange.cs

```diff
@@ -0,0 +1,14 @@
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    public class HiDataRange
+    {
+        public string From { get; }
+        public string To { get; }
```

Comment:
Why can't it be datetime?

Reply:
It is stored as a string in consent manager